modules/metastorage-server/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDBKeyValueStorage.java

```diff
@@ -208,14 +213,16 @@ public RocksDBKeyValueStorage(Path dbPath) {
      * @throws RocksDBException If failed.
      */
     private void destroyRocksDB() throws RocksDBException {
-        try (final Options opt = new Options()) {
+        try (Options opt = new Options()) {
             RocksDB.destroyDB(dbPath.toString(), opt);
         }
     }
 
     /** {@inheritDoc} */
     @Override public void close() throws Exception {
-        IgniteUtils.closeAll(options, data, index, db);
+        IgniteUtils.closeAll(data, index, db, options);
+
+        snapshotExecutor.shutdown();
```

Comment:
this is not a completely correct way to shutdown an executor, please use `IgniteUtils#shutdownAndAwaitTermination`. You should also place it before the `closeAll` call in order to let the background tasks to finish